### PR TITLE
Initial implementation of core service

### PR DESCRIPTION
**Pull request by Ethanljf**
New "Core" service which performs periodic updates to summary data and guidance.

### CODE REVIEW

File: services/core/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:20.04
+
+# Copy local code to the container image.
+ENV HOME /core
+WORKDIR $HOME
+COPY . ./
+
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends apt-utils git && \
```
Comment (tparrott-cse) [inline services/core/Dockerfile:9]: See the other Dockerfiles for the new convention that should be applied here...

**Review by tparrott-cse — COMMENTED** [review]
Please see my inline comments

**Comment by Ethanljf** [comment]
> Please see my inline comments

Updated

**Review by tparrott-cse — APPROVED** [review]

**Review by nsdeschenes — APPROVED** [review]

**Pull request merged**